### DISCUSSION TITLE
Micro trends?

### DISCUSSION THREAD

**Discussion by 1st-class-netizen** [mikedilger/gossip#910] [Ideas]
This is not a request for a micro app. I kind of want an extra UI element for trends, even if the trends just maps who I follow (in large part from iris.to's trending, but that is above and beyond a mere client I would think) into a micro trends. The micro trends could be subdivided into "popular trending posts" and "popular hashtag".

**Comment by mikedilger** [comment]
That is a reasonable thing to want.  But I'm not sure gossip is the right place.  I use multiple nostr apps depending on what kind of thing I'm interested in looking at.  I think other nostr apps/services already do this and if not, a new website showing trends would be my preference.

Gossip is trying to hone fairly tightly to being a twitter-like client, and I'm trying to avoid any features that go outside of that scope at least until I get all the planned features completed (I broke that rule a few times, like doing DMs and being a bunker, but there were other reasons why those made sense).

But if you wrote such code and offered it in a PR and it worked, a separate page perhaps in the UI, I would probably merge it.

**Comment by 1st-class-netizen** [comment]
You're kind of right to say that Twitter doesn't have micro trends. But you can't blame me for trying to split the difference can you? While I don't think I can contribute here concerning Rust and LMMD knowledge, I think it was nice to get a temperature take on how this Twitter feature might graduate into the Twitter-like sphere when it comes to functioning in a client context rather than a platform one. Seeing the lay of the land on that, I guess this shows everyone how to plan ahead in their use of Nostr apps.
